youtube_dlc/downloader/youtube_live_chat.py

```diff
@@ -28,41 +31,26 @@ def real_download(self, filename, info_dict):
             'total_frags': None,
         }
 
+        ie = YT_BaseIE(self)
```

Comment:
```suggestion
        ie = YT_BaseIE(self.ydl)
```

Despite being called `downloader`, the actual argument it expects is the `YoutubeDL` object

Reply:
Realized afterwards that I could have just pressed the button in the suggestion :sweat_smile: